ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandIdAssigner.java

```diff
@@ -142,6 +145,14 @@ private static CommandId getAlterSourceCommandId(final AlterSource alterSource)
     );
   }
 
+  private static CommandId getAlterSystemCommandId(final AlterSystemProperty alterSystemProperty) {
+    return new CommandId(
+        Type.CLUSTER,
+        alterSystemProperty.toString(),
```

Comment:
why are you using the string form of alter system property here?

Reply:
I don't think it will parse correctly when the command id is serialized/deserialized because the command id always serializes to something like "<type>/<name>/<action>". In this case type is CLUSTER and action is ALTER. But the name is just the stringified version of AlterSystemProperty. We can't really come up with a name for this operation. Can you just use a uuid here to ensure it's unique?